packages/elements/tests/snapshots/dds-link-with-icon.md

```diff
@@ -0,0 +1,40 @@
+# `dds-link-with-icon`
```

Comment:
Just checking, is this file generated from snapshot tests? Wondering if this can be output into a snapshots folder under the corresponding component folder instead?

Reply:
Good question, yes to the former. Per-component snapshot directory seems to be hard: https://www.npmjs.com/package/karma-snapshot#config